src/biguint.rs

```diff
@@ -2689,6 +2689,34 @@ impl BigUint {
         let zeros: u64 = self.data[i].trailing_zeros().into();
         Some(i as u64 * u64::from(big_digit::BITS) + zeros)
     }
+
+    /// Returns the number of significant bits: bit position
+    /// of leftmost 1-bit + 1, or 0 for no 1-bits (number is 0).
+    pub fn significant_bits(&self) -> u64 {
+        if let Some(i) = self.data.iter().rev().position(|&digit| digit != 0) {
+            let i = self.data.len() - i - 1;
+            let zeros: u64 = self.data[i].leading_zeros().into();
+            (i + 1) as u64 * u64::from(big_digit::BITS) - zeros
+        } else {
+            0
+        }
+    }
+
+    #[cfg(feature = "trailing_ones")]
+    /// Returns the number of least-significant bits that are ones.
+    pub fn trailing_ones(&self) -> u64 {
+        if let Some(i) = self.data.iter().position(|&digit| !digit != 0) {
+            let ones: u64 = self.data[i].trailing_ones().into();
```

Comment:
The `std` implementation of `trailing_ones` for primitives is literally just `(!self).trailing_zeros()`, so I think you can do the same here to avoid the feature / question of rustc support.
```suggestion
            let ones: u64 = (!self.data[i]).trailing_zeros().into();
```